lib/run-query.js

```diff
@@ -154,7 +155,10 @@ clients.postgres = function (query, connection, queryResult, callback) {
     password: connection.password,
     database: connection.database,
     host: connection.host,
-    ssl: connection.postgresSsl
+    ssl: connection.postgresSsl ? {
```

Comment:
I'm not entirely certain, but I think we still need to support ssl: true here. Setting to true without passing in any cert or key will use ssl but not alter any ssl parameters... (I think anyways - I've only set this once a long time ago when querying heroku postgres)